yt_dlp/extractor/arte.py

```diff
@@ -136,7 +126,9 @@ def _real_extract(self, url):
         lang = mobj.group('lang') or mobj.group('lang_2')
         langauge_code = self._LANG_MAP.get(lang)
 
-        config = self._download_json(f'{self._API_BASE}/config/{lang}/{video_id}', video_id)
+        config = self._download_json(f'{self._API_BASE}/config/{lang}/{video_id}', video_id, headers={
+            "x-validated-age": 18
```

Comment:
Single quotes
And core code fixes this, but all header values should be strings
```suggestion
            'x-validated-age': '18'
```